extensions/brew/package.json

```diff
@@ -39,6 +40,13 @@
       "subtitle": "Brew",
       "description": "Upgrade brew formulae & casks",
       "mode": "no-view"
+    },
+    {
+      "name": "clean-up",
+      "title": "Clean Up",
+      "subtitle": "Brew",
+      "description": "Clean Up Everything At Once",
```

Comment:
Better just use "Sentence case" here (uppercase just the first word) 😄

Reply:
Thanks... & I fix that